service/src/componentTest/resources/com/integreety/activityengine/step/find-activity.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Find activity
+
+  Scenario: find an activity by id
+  Given an existing activity with lessonId ABC
+  When the activity is requested by the returned id
```

Comment:
not sure I follow what this means - which returned id?: `Given the activity is requested by the returned id`

Reply:
Yes, I see how confusing this is. This is the id that was generated in the Given when the activity was saved. I have amended the working.